src/ec_plists.erl

```diff
@@ -886,7 +871,6 @@ runmany_wrap(Fun, Parent) ->
         throw:R:Stacktrace ->
             Parent ! {erlang:self(), error, {{nocatch, R}, Stacktrace}}
     end.
```

Comment:
This one definitely needs to stay around.

Reply:
Sure.